### ISSUE TITLE
[Automation] Script for automating Onboarding to Cluster 

### ISSUE THREAD

**Issue by HumairAK** [operate-first/apps#635]
The steps [here](https://www.operate-first.cloud/users/support/docs/onboarding_to_cluster.md) should be automated within a script. Either we can extend the current [onboarding script](https://github.com/operate-first/apps/blob/master/scripts/onboarding.sh) used as one of the steps to handle the whole procedure, or create a new one. The tricky part would be handling pgp keys. 

I'm imagining the usage would look something like this: 

```
$ ./onboard_to_cluster.sh [team_name] [list_of_namespaces] [list_of_users] [project_description] [env] [cluster] -o <optional_pgp_key>
```

Maybe it would be better to require a yaml file with a set schema like: 

```yaml
onboarding_template:
  team-name: example_team
  list-of-namespaces:
    - ns_1
    - ns_2
  list-of-users:
    - user_1
    - user_2
  project_description: "example description" 
  env: MOC
  cluster: ZERO
  optional_pgp_keys:
    - key_1
    - key_2
```

And the script can just consume this file: 

```bash
$ ./onboard_to_cluster.sh onboard.yaml
```

Then a next step can be to eventually have people fill out a yaml/form and have a bot perform the PR.

**Comment by tumido** [comment]
I think for the UX it would be better to support both of these 2 options:

1. Declarative - like the YAML you've mentioned
2. Interactive - like a wizard (imagine `npm init` or [inquirer](https://www.npmjs.com/package/inquirer) that I know from the JS world - I'd prefer to have this in golang in the end though...)

It think this ties back to the ultimate CLI idea we brainstormed some time ago (with no resolution as far as I remember). Ideally it would be nice to have an interactive/declarative wrapper around all these different tasks we'd like to automate.

**Comment by HumairAK** [comment]
one issue I'm seeing is right now we have a few scripts in bash, one in python (that @larsks  has worked on) we will need to narrow down to one language if we are to wrap these, lest we start calling bash / python scripts from golang or something (yuck).

**Comment by HumairAK** [comment]
also tagging @ipolonsk as she has been helping with automating

**Comment by larsks** [comment]
@HumairAK I think we should open a separate issue for that discussion.

**Comment by HumairAK** [comment]
depends on: https://github.com/operate-first/apps/issues/637
